evolver/calibration/interface.py

```diff
@@ -193,5 +193,5 @@ class Config(Calibrator.Config):
         input_transformer: dict[int, ConfigDescriptor | Transformer | None] | None = None
         output_transformer: dict[int, ConfigDescriptor | Transformer | None] | None = None
 
-    def create_calibration_procedure(self, selected_hardware, *args, **kwargs):
+    def create_calibration_procedure(self, selected_hardware, resume, *args, **kwargs):
```

Comment:
I know this is not added in this PR, but I don't think we even need this method here. It is abstract in the base and we can leave it out to have consistent error (`TypeError` from the ABC) - plus satisfying to remove code!

Reply:
Ok, cool will fix.